File: gaurav2java/leapyear.java
import java.util.*;
public class leapyear {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the Year:");
        int x = sc.nextInt();
        if(x%4 ==0)
        System.out.println("The Year of Value Is Leap Year");
        else
        System.out.println("The Year of Value Is Not Leap Year ");
        sc.close();
    }
    

}

File: C PROGRAMMING/conditional.c/leapyear.c
#include <stdio.h>
int main()
{
    int x;
    printf("Enter The Year of x:");
    scanf("%d", &x);
    if (x % 400 == 0 || (x%4==0 && x%100!=0))
        printf("Leap Year\n");
    else
  printf("Not leap Year\n");

    return 0;
}